app/models/concerns/order_status.rb

```diff
@@ -103,7 +126,7 @@ def update_status(status, params = {})
     send(status)
   end
 
-  APPROVED_STATUSES = %w(approved filled shipped received closed).map(&:freeze).freeze
+  APPROVED_STATUSES = %w(approved filled shipped received closed canceled).map(&:freeze).freeze
```

Comment:
It seems like `canceled` status sort of straddles the line of approved and not approved since it could come from non-approved statuses. Does having it in the `APPROVED_STATUSES` add some benefits? Should there be a separate state for canceled from before versus after approval? That seems like that might be ugly though.

Reply:
Looks like like revealed some unused code. Also looks like the only place this impacts is report value_by_county throught the Org.approved_orders association. I am removing canceled since I think this is basically like a massive undo button to flush out bad orders. 

they should not show or reports or anything. 

What do you think about removing them from the default scope?